### PR TITLE
Update validator action version and turn off dev deps installation

### PR DESCRIPTION
**Pull request by knightdave**
fix https://github.com/insightsengineering/idr-tasks/issues/211

### CODE REVIEW

**Review by dinakar29 — APPROVED** [review]
LGTM

File: .github/workflows/validation.yaml [hunk 1]
```diff
@@ -53,7 +53,9 @@ jobs:
           SD_ENABLE_CHECK: ${{ inputs.enable-staged-dependencies-check }}
 
       - name: Build report 🏗
-        uses: insightsengineering/thevalidatoR@v1.1.1
+        uses: insightsengineering/thevalidatoR@v1.1.2
```
Comment (dinakar29) [inline .github/workflows/validation.yaml:56]: Technically 1.2.0, since you added a new feature, no?

Comment (knightdave) [reply .github/workflows/validation.yaml:56]: It depends, basically tool functionality didn't change - adding new variable was related to fixing bug

Comment (dinakar29) [reply .github/workflows/validation.yaml:56]: Adding a new flag/feature would induce a minor version upgrade, if we're to follow semantic versioning standards, even if it comes with a bugfix.

In the release notes, we'd include 2 items - the bugfix as well as the new flag/variable/setting.

**Pull request merged**